samples/bluetooth/hci_uart/README.rst

```diff
@@ -125,3 +125,19 @@ application. To enable debug over RTT the debug configuration file can be used.
    west build samples/bluetooth/hci_uart -- -DOVERLAY_CONFIG='debug.conf'
 
 Then attach RTT as described here: :ref:`Using Segger J-Link <Using Segger J-Link>`
+
+Support for the Direction Finding
```

Comment:
This is a very limited information about DF. I suggest to add here more comprehensive information or reference to direction finding samples. 
Or change this to information about DTM mode that allows to test DF functionality with V3/V4 version of HCI test commands.

Reply:
Ok I added a references to the Direction Finding samples